uaclient/cli/vulnerability/show.py

```diff
@@ -0,0 +1,275 @@
+                    "NAME",
+                    "CURRENT VERSION",
+                    "FIXED BY",
+                    "FIX AVAILABLE FROM",
+                ],
+                rows=[
+                    [
+                        pkg.name,
+                        pkg.current_version,
+                        pkg.fix_version,
+                        pkg.fix_available_from,
+                    ]
+                    for pkg in vulnerability_issue.affected_packages
+                ],
+            )
+        )
+        + "\n"
+    )
+
+
+def _show_related_usns(related_usns):
+    print(
+        messages.CLI_VULNERABILITY_SHOW_RELATED_USNS
+        + str(
+            Table(
+                headers=[
+                    "VULNERABILITY",
+                    "TITLE",
+                    "AFFECTED INSTALLED PACKAGES",
+                ],
+                rows=[
+                    [
+                        usn.name,
+                        usn.title,
+                        ", ".join(usn.affected_installed_packages),
+                    ]
+                    for usn in related_usns
+                ],
+            )
+        )
+        + "\n"
+    )
+
+
+def _show_related_cves(related_cves):
+    print(
+        messages.CLI_VULNERABILITY_SHOW_RELATED_CVES
+        + str(
+            Table(
+                headers=[
+                    "VULNERABILITY",
+                    "PRIORITY",
+                    "AFFECTED INSTALLED PACKAGES",
+                ],
+                rows=[
+                    [
+                        cve.name,
+                        vuln_util.get_priority_with_color(cve.priority),
+                        ", ".join(cve.affected_installed_packages),
+                    ]
+                    for cve in related_cves
+                ],
+            )
+        )
+        + "\n"
+    )
+
+
+def _show_cve_info(cfg: config.UAConfig, cve: str, data_file: Optional[str]):
+    cve_vulnerabilities_result = cve_vulnerabilities(
+        options=CVEVulnerabilitiesOptions(all=True, data_file=data_file),
+        cfg=cfg,
+    )
+
+    cve_data = None
+    for cve_info in cve_vulnerabilities_result.cves:
+        if cve_info.name == cve:
+            cve_data = cve_info
+            break
+
+    if not cve_data:
+        print(messages.CLI_VULNERABILITY_SHOW_NOT_AFFECTED.format(issue=cve))
+        return
+
+    _show_base_info(cve_vulnerabilities_result, cve_data)
+    print(
+        messages.CLI_VULNERABILITY_SHOW_UBUNTU_PRIORITY.format(
+            cve_info.ubuntu_priority
+        )
+    )
+    print(
+        messages.CLI_VULNERABILITY_SHOW_CVSS_SCORE.format(cve_info.cvss_score)
+    )
+    print(
+        messages.CLI_VULNERABILITY_SHOW_CVSS_SEVERITY.format(
+            cve_info.cvss_severity
+        )
+    )
+    _show_description(cve_data)
+    _show_cve_affected_packages(cve_data)
+
+    if cve_data.notes:
+        print(messages.CLI_VULNERABILITY_SHOW_NOTES)
+
+        for note in cve_data.notes:
```

Comment:
Newlines are weird between the notes and the related USNs.
```
$ pro vulnerability show CVE-2024-8088
(...)
[NOTES]

mdeslaur> The original fix lead to a regression, see subsequent commit.
There are two more commits in the Lib/zipfile directory that are
possibly related to this issue.
[RELATED USNs]
VULNERABILITY  TITLE                   AFFECTED INSTALLED PACKAGES
USN-7015-1     Python vulnerabilities  python3.10
```

You see `[NOTES]` is the only header which has a newline before the content, and there is no newline before `[RELATED USNs]`. The problem seems to be in the notes block, as a CVE without notes is formatted correctly.